requirements.txt

```diff
@@ -0,0 +1 @@
+pyusb==1.0.2
```

Comment:
Recommend `>=` rather than pinning to 1.0.2 (that version plus a later version work worked for me).